DetourCrowd/Source/DetourCrowd.cpp

```diff
@@ -386,7 +386,7 @@ bool dtCrowd::init(const int maxAgents, const float maxAgentRadius, dtNavMesh* n
 	m_maxAgents = maxAgents;
 	m_maxAgentRadius = maxAgentRadius;
 
-	dtVset(m_ext, m_maxAgentRadius*2.0f,m_maxAgentRadius*1.5f,m_maxAgentRadius*2.0f);
+	dtVset(m_halfExtents, m_maxAgentRadius*2.0f,m_maxAgentRadius*1.5f,m_maxAgentRadius*2.0f);
```

Comment:
It's intentional. Better name might be a good idea. If nothing else `m_agentPlacementHalfExtents`.

The intent is that it defines how far off the agent can be outside the navmesh to be "snapped" back to navmesh in case of shit-happens. Two reasons it is small. I wanted the agents to get stuck/stop when too far outside navigable area, and secondly large query extents is expensive to process.